<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LTCH-B3J24253GP" failures="1" tests="1" name="SauceLabTest" time="0.078" errors="0" timestamp="2024-12-08T11:51:41 IST">
  <testcase classname="cucumber.api.testng.AbstractTestNGCucumberTests" name="@BeforeClass setUpClass" time="0.016">
    <failure type="java.lang.NoClassDefFoundError" message="cucumber/runtime/CucumberException">
      <![CDATA[java.lang.NoClassDefFoundError: cucumber/runtime/CucumberException
at cucumber.api.testng.AbstractTestNGCucumberTests.setUpClass(AbstractTestNGCucumberTests.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.ClassNotFoundException: cucumber.runtime.CucumberException
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
... 19 more
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass setUpClass -->
  <testcase classname="cucumber.api.testng.AbstractTestNGCucumberTests" name="@AfterClass tearDownClass" time="0.002">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;cucumber.api.testng.TestNGCucumberRunner.finish()&amp;quot; because &amp;quot;this.testNGCucumberRunner&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "cucumber.api.testng.TestNGCucumberRunner.finish()" because "this.testNGCucumberRunner" is null
at cucumber.api.testng.AbstractTestNGCucumberTests.tearDownClass(AbstractTestNGCucumberTests.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass tearDownClass -->
  <testcase classname="cucumber.api.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.0">
    <failure type="java.lang.RuntimeException" message="java.lang.NullPointerException: Cannot invoke &amp;quot;cucumber.api.testng.TestNGCucumberRunner.provideScenarios()&amp;quot; because &amp;quot;this.testNGCucumberRunner&amp;quot; is null">
      <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException: Cannot invoke "cucumber.api.testng.TestNGCucumberRunner.provideScenarios()" because "this.testNGCucumberRunner" is null
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.NullPointerException: Cannot invoke "cucumber.api.testng.TestNGCucumberRunner.provideScenarios()" because "this.testNGCucumberRunner" is null
at cucumber.api.testng.AbstractTestNGCucumberTests.scenarios(AbstractTestNGCucumberTests.java:31)
... 19 more
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- SauceLabTest -->
